### ISSUE TITLE
ZFS encryption: support for zfs send -w

### ISSUE THREAD

**Issue by rsplaul** [jgoerzen/simplesnap#25]
Now that ZFS on linux supports encryption (even in outdated versions provided by distros), I think it would be advisable that simplesnap suported the `zfs send -w/--raw` command line option.

- It would be possible to do encrypted backups of encrypted datasets on untrusted backuphosts, because the backuphost doesn’t have to know the key.
- It would be possible to backup datasets for which the key hasn’t been loaded (yet).
- Even for trusted environments the backuphost doesn’t have to do encryption for the backups itself. (Faster backups on Raspberry Pi and the like!)

This raw send mode should however be configurable because a user might want an unencrypted backup of their encrypted datasets. Also, `zfs send -w` doesn’t work for incremental backups on previous backups created without this option.

**Comment by jgoerzen** [comment]
You are right.  Actually we need a more general way to set send options like -e, -c, etc.  It will be a bit before I have more time to give to simplesnap, but it's on the list.
